src/internal/_symbolIterator.js

```diff
@@ -0,0 +1 @@
+module.exports = (typeof Symbol !== 'undefined') ? Symbol.iterator : '@@iterator';
```

Comment:
do all symbol implementation s expose iterator? I know a lot of them don't include the es6 symbol.match yet